PhotoNim.nim

```diff
@@ -1,10 +1,12 @@
 let PhotoNimVersion* = "PhotoNim 0.2"
 
-import src/[geometry, pcg, hdrimage, scene, material, hitrecord, camera]
-export geometry, pcg, hdrimage, scene, material, hitrecord, camera
+import src/[geometry, pcg, hdrimage, scene, material, hitrecord, camera, sceneFiles]
```

Comment:
I would better arrange code in the src/ directory in v0.3.1

Reply:
Usted ya sabe